src/pages/fieldManagement/settings/defaultFieldComponents/RelationshipEditor.jsx

```diff
@@ -0,0 +1,304 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { get, filter } from 'lodash-es';
+
+import { FormattedMessage } from 'react-intl';
+import IconButton from '@material-ui/core/IconButton';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import NewChildIcon from '@material-ui/icons/AddCircle';
+import TextInput from '../../../../components/inputs/TextInput';
+import DeleteButton from '../../../../components/DeleteButton';
+import Button from '../../../../components/Button';
+import Text from '../../../../components/Text';
+
+const intl = useIntl();
```

Comment:
For me, this line is preventing the hot updater from working.
```
Invalid hook call. Hooks can only be called inside of the body of a function component.
```

Reply:
Done